tests/unit/test_adapter.py

```diff
@@ -541,3 +541,12 @@ def test_convert_date_type(self):
         expected = ["date", "date", "date"]
         for col_idx, expect in enumerate(expected):
             assert AthenaAdapter.convert_date_type(agate_table, col_idx) == expect
+
+    def test_parse_s3_path(self):
```

Comment:
- I think you can use pytest fixtures for this
- This test should be in class `TestAthenaAdapter` and not `TestAthenaAdapterConversions`

Reply:
> - This test should be in class TestAthenaAdapter and not TestAthenaAdapterConversions

🤦 

> - I think you can use pytest fixtures for this

Is that what you meant, or did you mean using some existing fixutre?

Reply:
I meant instead of doing a for loop inside your tests, you can decorate your function with `@pytest.fixtures` and it will create 2 tests. There are examples in this test file if you want :)

Reply:
How does that look now?

Reply:
Great 👍🏻